acquisitions/src/main/resources/karate-config.js

```diff
@@ -118,16 +118,17 @@ function fn() {
       password: 'admin'
     }
   } else if (env == 'rancher') {
-    config.baseUrl = 'https://thunderjet-okapi.ci.folio.org';
+    config.baseUrl = 'https://folio-dev-thunderjet-okapi.ci.folio.org';
     config.edgeUrl = 'https://folio-snapshot.dev.folio.org:8000';
     config.ftpUrl = 'ftp://ftp.ci.folio.org';
     config.ftpPort = 21;
     config.ftpUser = 'folio';
     config.ftpPassword = 'Ffx29%pu';
+    config.prototypeTenant= 'mobius'
     config.admin = {
-      tenant: 'diku',
-      name: 'diku_admin',
-      password: 'admin'
+      tenant: 'mobius',
+      name: 'mobius_admin',
+      password: 'mobius_admin'
```

Comment:
I don't think we can use mobius tenant for all integration tests on rancher, because consortia theretically can not be enabled on racnher. I suggest to introduce "thunderjet-consortia-rancher" env branch and move this config here

Reply:
fixed